### ISSUE TITLE
Can't drag an entire cell

### ISSUE THREAD

**Issue by kylesluder** [HappenApps/Quiver#183]
If this is something you can do, I can't seem to figure out how.

**Comment by ylian** [comment]
No, you can't. Maybe in the future. Right now you can only use keyboard shortcuts (look under Cell Menu).

**Comment by yaodong** [comment]
It's better if I can drag & drop cells to reorder them. It will be very convenient for writing.

**Comment by aamnah** [comment]
+1 for drag and drop cell arrangement. Specially convenient when you have multiple types of cells

**Comment by habitualshaker** [comment]
+1 for this

**Comment by ghost** [comment]
+1 for this feature as well

**Comment by ronlipke** [comment]
+1 for this.

**Comment by maxiappleton** [comment]
+1 would be a great feature/functionality to add

**Comment by KimMichaelJones** [comment]
+1 for this as well, quite painful without it

**Comment by jxshen311** [comment]
+1 for this feature as well
